submit_and_compare/tests/test_all.py

```diff
@@ -7,7 +7,6 @@
 
 from unittest import mock
 from django.test.client import Client
-from django.utils.translation import ugettext as _
```

Comment:
Great! `ugettext` makes no sense in testing. At least, wasn't a good way to test i18n.